{{cookiecutter.project_slug}}/src/js/utils/api.js

```diff
@@ -46,7 +46,7 @@ const getApiFetch = (onAuthFailure: () => void) => (
           onAuthFailure();
           return getResponse(response);
         }
-        if (response.status === 404) {
+        if (response.status === 403 || response.status === 404) {
```

Comment:
Maybe this should actually be `response.status >= 400 && response.status < 500`?

Reply:
Fair point. 👍

Reply:
Hm, actually I'm not so sure. The rest of the status codes in the 400 range seem like errors we want to throw and log, whereas these (403 and 404) just represent "not found" in my mind.